pageserver/src/tenant.rs

```diff
@@ -2359,6 +2359,25 @@ impl Tenant {
 
         size::gather_inputs(self, logical_sizes_at_once, &mut shared_cache).await
     }
+
+    /// Calculate synthetic tenant size
+    /// This is periodically called by background worker
+    ///
+    #[instrument(skip_all, fields(tenant_id=%self.tenant_id))]
+    pub async fn calculate_synthetic_size(&self) -> anyhow::Result<u64> {
+        let inputs = self.gather_size_inputs().await?;
+
+        let size = inputs
+            .calculate()
+            .unwrap_or_else(|e| panic!("err {}, inputs {:?}", e, inputs)); // FIXME this panic is debug only.
```

Comment:
We could move this panic into the error message...?
```suggestion
            .with_context(|e| format!("tenant size model calculation failed for {inputs:?}"));
```